recipes-openxt/xenclient-dom0-tweaks/xenclient-dom0-tweaks/openxt.cfg

```diff
@@ -5,31 +5,31 @@ sinit=GM45_GS45_PM45_SINIT_51.BIN Q35_SINIT_51.BIN Q45_Q43_SINIT_51.BIN i5_i7_DU
 ucode=microcode_intel.bin
 
 [openxt-normal]
-options=placeholder console=com1 dom0_mem=min:420M,max:420M,420M efi=no-rs,attr=uc com1=115200,8n1,pci mbi-video vga=current flask=enforcing loglvl=debug guest_loglvl=debug smt=0 ucode=-1
+options=placeholder console=com1 dom0_mem=min:420M,max:420M,420M efi=rs,attr=uc com1=115200,8n1,pci mbi-video vga=current flask=enforcing loglvl=debug guest_loglvl=debug smt=0 ucode=-1
```

Comment:
At least one system hanged at boot on EFI runtime services calls (HP EliteDesk 800 G1) ([OXT-1377](https://openxt.atlassian.net/browse/OXT-1377)).

Reply:
Just to clarify, the 800 G1 still hangs even with the rest of the x64 stuff?